data/en/isstruct.json

```diff
@@ -17,6 +17,22 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+		"title":"isStruct Example",
+		"description":"Returns true if variable is a ColdFusion structure or is a Java object that implements the java.lang.Map interface. ",
+		"code":"someStruct=StructNew(\"Ordered\");\r\nisStruct(someStruct)",
```

Comment:
Something like this would also suffice: `isStruct( structNew() )` or `isStruct(cgi)`